lib/ansible/modules/net_tools/ipwcli_dns.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Christian Wollinger <cwollinger@web.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ipwcli_dns
+
+short_description: Manage DNS Records for Ericsson IPWorks via ipwcli
+
+version_added: "2.10"
+
+description:
+    - "Manage DNS records for the Ericsson IPWorks DNS server. The module will use the ipwcli to deploy the DNS records."
+
+notes:
+    - To make the DNS record change affective, you need to run 'update dnsserver' on the ipwcli.
+
+options:
+    dnsname:
+        description:
+            - Name of the record.
+        required: true
+        type: str
+    type:
+        description:
+            - Type of the record.
+        required: true
+        type: str
+        choices: [ NAPTR, SRV, A ]
```

Comment:
Are these the only choices? E.g. is `TXT` or `AAAA` not permitted? If they are permitted, the choices might better be left out?

Reply:
At the moment the module only supports A, SRV and NAPTR

Reply:
I consider that a strong limitation; isn't the world going to AAAA ?

Reply:
Will add this